crypto/crypto.go

```diff
@@ -41,8 +41,9 @@ var kidPattern = regexp.MustCompile(`^[\da-zA-Z_\- :#.]+$`)
 
 // Config holds the values for the crypto engine
 type Config struct {
-	Storage string              `koanf:"storage"`
-	Vault   storage.VaultConfig `koanf:"vault"`
+	Storage       string                  `koanf:"storage"`
+	Vault         storage.VaultConfig     `koanf:"vault"`
+	StorageClient storage.APIClientConfig `koanf:"connection"`
```

Comment:
wouldn't `storage` be nicer?

`crypto.storage.whatever`

Reply:
`storage` is already in use.